docker-compose.yml

```diff
@@ -1,17 +1,20 @@
-web:
-  image: colinxfleming/dcaf_case_management
-  command: rails s -p 3000 -b 0.0.0.0
-  volumes:
-    - .:/usr/src/app
-  ports:
-    - "3000:3000"
-  links:
-    - db
-  environment:
-    mongohost: db
-  env_file:
-    - web-variables.env
-db:
-  image: mongo
-  ports:
-    - "27017:27017"
+version: '3.0'
```

Comment:
CircleCI is, unfortunately, a few docker versions behind and I worry this'll bust the CI build until they get their act together and upgrade (which is in progress, I hear). It is with a heavy heart that I must ask to revert this change :(